Gemfile

```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 
+git_source(:avant_repo) { |repo| "https://github.com/avantcredit/#{repo}.git" }
+
+gem 'stathat-json', '~> 0.1', avant_repo: 'stathat-json'
```

Comment:
is it possible to put this in the `gemspec` file rather than here, or can you not specify the source repo in that case? Convention is to put dependencies in the gemspec: https://github.com/avantcredit/event_emitter/blob/master/avant-event_emitter.gemspec#L30

Reply:
@jlovell I understand the convention -- a quick Google search didn't yield a way to specify sources in the `gemspec`, if someone know how I'm open to changing it

Reply:
@jlovell I saw [this post](https://groups.google.com/d/msg/ruby-bundler/U7FMRAl3nJE/1KIUJTmuzzQJ) and changed the code to be something similar

Reply:
Not sure if it's what we want though -- open to suggestions